### ISSUE TITLE
5.5.0 Regression: Issue with testing of stripe card component

### ISSUE THREAD

**Issue by juliachruszczowa** [cypress-io/cypress#8992]
After upgrading to version 5.5.0 there is an issue with testing of stripe card component

### Current behavior
![Selection!UNITO-UNDERSCORE!568](https://user-images.githubusercontent.com/41747580/97364790-d8723a80-18a4-11eb-918c-b683d41bfec5.png)
in cypress:open mode credit card field is not displayed

### Versions
Works correctly in version 5.4.0
![Selection!UNITO-UNDERSCORE!569](https://user-images.githubusercontent.com/41747580/97364897-0eafba00-18a5-11eb-86a0-3984c7d1265b.png)

**Comment by jennifer-shehane** [comment]
Right now there doesn't seem to be enough information to reproduce the problem on our end. We'll have to close this issue until we can reproduce it. This does not mean that your issue is not happening - it just means that we do not have a path to move forward. 

Please comment in this issue with a reproducible example and we will consider reopening the issue. Here are some tips for providing a [Short, Self Contained, Correct, Example](http://sscce.org/) and our own [Troubleshooting Cypress](https://on.cypress.io/troubleshooting) guide.

**Comment by juliachruszczowa** [comment]
@jennifer-shehane 
to reproduce the issue this example simple test can be run on page https://stripe.dev/elements-examples/

`describe('My First Test', function () {
  it('test', function () {
    cy.visit('https://stripe.dev/elements-examples/');

    cy.get('.__PrivateStripeElement > iframe')
      .eq(0)
      .then(($iframe) => {
        const body = $iframe.contents().find('.InputContainer');
        cy.wrap(body).find(`input[name="cardnumber"]`).type('4242424242424242');

        cy.wrap(body).find(`input[name="exp-date"]`).type('12/20');

        cy.wrap(body).find(`input[name="cvc"]`).type('424');
      });
  });
});`

on version 5.4.0 it is working: 
![Selection_570](https://user-images.githubusercontent.com/41747580/97400375-4ba09e80-18ef-11eb-9a35-418d34ff94cb.png)

on version 5.5.0 - not: 
![Selection_571](https://user-images.githubusercontent.com/41747580/97400408-5d824180-18ef-11eb-8d0a-b0e808c743ab.png)

**Comment by jennifer-shehane** [comment]
Thanks for providing the example!

I can reproduce with the code below in Cypress 5.5.0. This was introduced in this PR: https://github.com/cypress-io/cypress/pull/8827. I git bisected to this commit where it started failing: https://github.com/cypress-io/cypress/commit/19fdf4306a6be1610af9d18467b053209c2d5229

```js
it('test', () => {
  cy.visit('https://stripe.dev/elements-examples/')
})
```

### 5.4.0

<img width="1229" alt="Screen Shot 2020-10-28 at 1 22 41 PM" src="https://user-images.githubusercontent.com/1271364/97402289-b4543f00-1920-11eb-9a63-9dfd9e6dba78.png">


### 5.5.0

<img width="1241" alt="Screen Shot 2020-10-28 at 1 12 02 PM" src="https://user-images.githubusercontent.com/1271364/97401424-30e61e00-191f-11eb-81cb-aaf0696e4965.png">

**Comment by juliachruszczowa** [comment]
@jennifer-shehane thanks a lot. I've already downgraded. 
Will wait for update.

**Comment by boardofchris** [comment]
This is a **significant** issue that impacts _all_ apps that use stripe. 

You only need to include stripe to trigger failures within cypress, you do need to interact with stripe on your page or test for cypress to fail. Some tests will work, others will fail with cypress errors depending on what stripe is doing in the background. 

I wasn't able to track this issue to cypress/stripe issue until I ran the failing tests in Firefox, where the error messages were more detailed. The embedded Electron browser fails with a generic cypress error. 

Everyone with  <script src="https://js.stripe.com/v3/"></script> in their project is impacted.

**Comment by fernandomartinez82** [comment]
I add more information. In my case I use 'Stripe' as a payment method. In version 5.1 - 5.4, works well. But with version 5.5 some field are blocked. And the text number card, date and ccv is not shown, only name of card appears.

![image](https://user-images.githubusercontent.com/70921622/97790707-96e5d600-1ba9-11eb-9235-b4ef02a3b0ca.png)

**Comment by jennifer-shehane** [comment]
@boardofchris @fernandomartinez82 We suggest downgrading to 5.4.0 until this is resolved. We are looking into a fix.

**Comment by fernandomartinez82** [comment]
> @boardofchris @fernandomartinez82 We suggest downgrading to 5.4.0 until this is resolved. We are looking into a fix.

@jennifer-shehane It's an option, but I have other problems with lower versions that are solved in version 5.5. I will create 2 different projects with 2 different versions of Cypress, until the error is resolved

**Comment by jennifer-shehane** [comment]
There's a PR open to fix this, so hopefully this makes it into the next release. https://github.com/cypress-io/cypress/pull/9018
